ValorantClient.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+import re
+import time
+from aiohttp import ClientSession
+import requests
+
+
+class ValorantClient(ClientSession):
```

Comment:
If you are inheriting from ClientSession, this would be more aptly named: ValorantClientSession